.ddev/commands/backstop/backstop

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+## Description: Run backstop visual regression tests
+## Usage: backstop [flags] [args]
+## Example: ddev backstop test --target=local --list=all
+
+backstop --config=/src/backstop.js "$@"
```

Comment:
When I run this, I get error below. I dont think this is related to my M1. Also, FYI M1 should work with latest Docker Desktop pref as per https://github.com/garris/BackstopJS/issues/1300#issuecomment-1424888520. Thats what I'm trying to test.

```
❯ ddev backstop --help
/bin/sh: 1: /mnt/ddev_config/commands/backstop/backstop: not found
Failed to run backstop --help: exit status 127
```

Reply:
I think your `.ddev/docker-compose.backstop.yaml` file hasn't been replaced with the new one in `.disabled-services/docker-compose.backstop.yaml`